src/ansible/inventory.yml

```diff
@@ -14,6 +14,9 @@ all:
         base_ipa:
           hosts:
             base-ipa
+        base_ipa2:
```

Comment:
why create base_ipa2?

Reply:
Fixed to use base_ipa only, this was an oversight on my part.